### ISSUE TITLE
Why should stop the gradient of u_hat?

### ISSUE THREAD

**Issue by LiangCVer** [bourdakos1/capsule-networks#9]
"u_hat_stopped = tf.stop_gradient(u_hat, name='stop_gradient')"

 I don't think W can be updated in that case
